packages/smarthr-ui/src/components/Dropdown/DropdownTrigger.tsx

```diff
@@ -13,16 +15,34 @@ import { tv } from 'tailwind-variants'
 
 import { tabbable } from '../../libs/tabbable'
 import { includeDisabledTrigger } from '../../libs/util'
+import { Tooltip } from '../Tooltip'
 
 import { DropdownContext } from './Dropdown'
 
-type Props = PropsWithChildren<ComponentProps<'div'>>
+type ConditionalWrapperProps = {
+  shouldWrapContent: boolean
+  wrapper: (children: ReactNode) => JSX.Element
+  children: ReactNode
+}
+
+/**
+ * 条件付きでラッパをレンダリングする
+ */
+export const ConditionalWrapper: FC<ConditionalWrapperProps> = ({
```

Comment:
```suggestion
const ConditionalWrapper: FC<ConditionalWrapperProps> = ({
```
見た感じDropdownTriggerの中でのみの利用なのでexportを取りたいです！

Reply:
ああっ、すみません移動したあとで残ってしまっていました😣。exportを取りました🙏
https://github.com/kufu/smarthr-ui/pull/5291/commits/01018a33124c1ae08982b9a9194093e34398b30c

Reply:
ありがとうございます〜！